cloudinit/distros/freebsd.py

```diff
@@ -13,16 +13,28 @@
 from cloudinit import subp, util
 from cloudinit.settings import PER_INSTANCE
 
+from .networking import BSDNetworking, NetworkConfig
+
 LOG = logging.getLogger(__name__)
 
 
+class FreeBSDNetworking(BSDNetworking):
```

Comment:
Let's put this class in `networking.py`. For now, I think it makes more sense there.

Reply:
Done